src/pages/people/People.tsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { makeStyles } from '@material-ui/core/styles';
 import { Container, Grid, Typography } from '@material-ui/core';
-import { ReactComponent as SVG } from '../svg/people.svg';
+import { ReactComponent as SVG } from '../../svg/people.svg';
```

Comment:
We could colocate SVGs to the /pages/people (for each) because we don't use any twice right?

Reply:
Very smart, I'll do that right now